### PR TITLE
Labels distribution

### PR DESCRIPTION
**Pull request by hbertrand**
Added a function that returns the distribution of labels.

### CODE REVIEW

File: ecodse_funtime_alpha/data.py [hunk 1]
```diff
@@ -3,23 +3,61 @@
...
+        Path to the csv file
+
+    Returns
+    -------
+        tf.data.ZipDataset
+            Elements of the dataset are structured (image, labels)
+    """
     samples = []
     with open(labels_csv) as csvfile:
         reader = csv.reader(csvfile, delimiter=',')
```
Comment (mgermain) [inline ecodse_funtime_alpha/data.py]: The way to skip the header is `reader.next()`, then enumerate and `if i` becomes useless

Comment (mgermain) [reply ecodse_funtime_alpha/data.py]: for `'{}.jpg'.format(row[0])` use fstring instead `f'{row[0]}.jpg'`

File: ecodse_funtime_alpha/data.py [hunk 2]
```diff
@@ -3,23 +3,61 @@
```
Comment (mgermain) [inline ecodse_funtime_alpha/data.py]: From improved readability do `from os import path`

File: tests/test_data.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 from PIL import Image
 import pytest
 
-from ecodse_funtime_alpha.data import get_dataset
+from ecodse_funtime_alpha.data import get_dataset, get_labels_distribution
```
Comment (mgermain) [inline tests/test_data.py]: one import per line
https://lintlyci.github.io/Flake8Rules/rules/E401.html

File: ecodse_funtime_alpha/data.py [hunk 4]
```diff
@@ -3,23 +3,61 @@
...
 
 
 def preprocess_image(image):
+    """
+    Decode a jpeg and normalize it to [0, 1] range.
+
+    Parameters
+    ----------
+    image : tf tensor
+        Raw image as a tf tensor
```
[the remark this answers is not in the document]
Comment (hbertrand) [reply ecodse_funtime_alpha/data.py]: No it doesn't, I just checked

**Comment by mgermain** [comment]
Awesome thanks (testing email reply)

File: ecodse_funtime_alpha/data.py (hunk 2 above)
Comment (hbertrand) [reply ecodse_funtime_alpha/data.py]: But I don't want to import path

File: ecodse_funtime_alpha/data.py (hunk 1 above)
Comment (hbertrand) [reply ecodse_funtime_alpha/data.py]: Done

File: tests/test_data.py (hunk 3 above)
Comment (hbertrand) [reply tests/test_data.py]: Done

File: ecodse_funtime_alpha/data.py [hunk 5]
```diff
@@ -57,18 +96,35 @@ def get_dataset(image_dir, labels_csv):
...
+    with open(labels_csv) as csvfile:
+        reader = csv.reader(csvfile, delimiter=',')
+        for i, row in enumerate(reader):
+            if i < 1:  # Skip first row as it is column names
+                continue
+            sample_labels.append(row[1].split(' '))
+
+    # Count labels occurences
+    labels = {}
+    for sample in sample_labels:
```
Comment (sblackburn86) [inline ecodse_funtime_alpha/data.py]: Why not use collections.Counter instead of manually looping through sample_labels?

Comment (hbertrand) [reply ecodse_funtime_alpha/data.py]: Done

**Pull request closed without merging**